editor/src/clj/editor/model.clj

```diff
@@ -619,7 +611,7 @@
   (output animation-set-build-target-single g/Any :cached produce-animation-set-build-target-single)
 
   (output pb-msg g/Any :cached produce-pb-msg)
-  (output save-value g/Any (gu/passthrough pb-msg))
+  (output save-value g/Any produce-save-value)
```

Comment:
I think we should add `:cached` on this `output` since it does quite a lot of work now?